salt/metalk8s/internal/sreport/installed.sls

```diff
@@ -1,7 +1,9 @@
+{%- from "metalk8s/macro.sls" import pkg_installed with context %}
```

Comment:
This diff should be squashed into the commit `Add salt states to install sos and inject custom plugins`, no?